openapi3.yaml

```diff
@@ -75,13 +186,31 @@ components:
           type: string
         description:
           type: string
-    anotherResource:
-      type: object
-      required:
-        - kind
-        - isAlive
-      properties:
-        kind:
+        bounding_polygon:
```

Comment:
Please use [`camelCase`](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#camel-case).
In general, be consistent in your naming conventions.

Reply:
ok, changed that ill update the branch.